README.md

```diff
@@ -51,4 +51,27 @@ You can still run Lighthouse under .NET Framework 4.6.1 if you wish. Clone this
 ### Examples of Lighthouse in the Wild
 Looking for some complete examples of how to use Lighthouse? Here's some:
 
-1. [Cluster.WebCrawler - webcrawling Akka.Cluster + Akka.Streams sample application.](https://github.com/petabridge/Cluster.WebCrawler)
\ No newline at end of file
+1. [Cluster.WebCrawler - webcrawling Akka.Cluster + Akka.Streams sample application.](https://github.com/petabridge/Cluster.WebCrawler)
+
+## Customizing Lighthouse / Avoiding Serialization Errors
+
+When using Akka.NET with extension modules like DistributedPubSub or custom serializers (like [Hyperion](https://github.com/akkadotnet/Hyperion)), you will get serialization errors logged.
```

Comment:
instead of "you will get serialization errors logged." say "serialization errors may appear in the logs because these modules are not installed and configured into Lighthouse by default"